public/data/saurabhmehta1601.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Saurabh Mehta",
+  "bio": "JavaScript Full Stack Developer",
+  "avatar":"https://github.com/saurabhmehta1601.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/saurabhmehta1601",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/saurabh__mehta",
+    "icon": "twitter"
```

*Suggested by @krishguptadev previously; just to bump and remind.*
```suggestion
      "icon": "twitter"
```